nautobot/core/views/mixins.py

```diff
@@ -232,6 +233,7 @@ class NautobotViewSetMixin(GenericViewSet, AccessMixin, GetReturnURLMixin, FormV
     # serializer_class has to be specified to eliminate the need to override retrieve() in the RetrieveModelMixin for now.
     serializer_class = None
     table_class = None
+    notes_form_class = NoteForm
```

Comment:
Why is this needed? I did a `git grep notes_form_class` and didn't see anywhere that it was being used - what'd I miss?

Reply:
It is for this test here
<img width="1327" alt="Screen Shot 2024-01-11 at 4 36 35 PM" src="https://github.com/nautobot/nautobot/assets/46973263/8e9af27a-7300-4439-953b-94305c84e253">

Reply:
Ah, that's why I didn't find it:

```python
            if getattr(self, f"{self.action}_form_class"):
                form_class = getattr(self, f"{self.action}_form_class")
```